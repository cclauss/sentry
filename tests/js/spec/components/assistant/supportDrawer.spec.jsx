import React from 'react';
import {shallow} from 'enzyme';
import SupportDrawer from 'app/components/assistant/supportDrawer';

describe('SupportDrawer', function() {
  it('renders', function() {
    let wrapper = shallow(<SupportDrawer onClose={() => {}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
