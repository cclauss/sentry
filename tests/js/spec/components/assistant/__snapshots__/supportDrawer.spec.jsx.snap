// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SupportDrawer renders 1`] = `
<StyledAssistantContainer
  hasResults={false}
>
  <StyledAssistantInputRow>
    <QuestionMarkIcon />
    <StyledSearchContainer>
      <StyledSearchIcon
        src="icon-search"
      />
      <StyledInput
        autoFocus={true}
        onChange={[Function]}
        placeholder="Search FAQs and docs..."
        type="text"
        value=""
      />
      <div
        className="close-button"
        onClick={[Function]}
        style={
          Object {
            "display": "flex",
          }
        }
      >
        <CloseIcon />
      </div>
    </StyledSearchContainer>
  </StyledAssistantInputRow>
</StyledAssistantContainer>
`;
